code/circuit.py

```diff
@@ -0,0 +1,24 @@
+class circuit:
```

Comment:
Usually, we name a class using upper-case letters, like so: `Circuit`. There's nothing wrong with naming the class `circuit`, though. Just be aware that it is different from what most Python code would do. 

If you choose to rename the class to `Circuit`, you _do not_ need to rename the file. Unlike Java, Python class names do not need to match the file name.